### PR TITLE
Update badlinks.ts

### PR DESCRIPTION
**Pull request by Angry-Pineapple3121**
made sure to remove ANYTHING related to sb drama (sbe, etc)
i also synced both repos together before pring so I dont leave out some links like last time
uh i hope this didnt break anything github shows 11.6k lines diff but you use 2tabs json spacing and i use 1 but i changed mine so idk i hope it doesnt break
anyways, enjoy

### CODE REVIEW

**Comment by Angry-Pineapple3121** [comment]
Checks / Formatting (pull_request) Failing after 27s — Formatting 

github sucks i hate this

**Comment by kr45732** [comment]
Y e s

**Review by kr45732 — APPROVED** [review]

**Comment by IRONM00N** [comment]
> Checks / Formatting (pull_request) Failing after 27s — Formatting
> 
> github sucks i hate this

my formatting workflow is just broken rn

**Comment by jani270** [comment]
can you please sort them

File: src/lib/badlinks.ts [hunk 1]
```diff
@@ -578,278 +725,509 @@ export default [
...
+	"discord-nitrogift.xyz",
 	"discord-nitros.com",
+	"discord-nitros.ru",
 	"discord-nitrot.xyz",
+	"discord-njtro.store",
 	"discord-nltro.com",
 	"discord-nltro.fun",
+	"discord-nltro.info",
 	"discord-nltro.ru",
+	"discord-nudes.club",
```
Comment (IRONM00N) [inline src/lib/badlinks.ts:954]: not even going to ask

**Review by IRONM00N — APPROVED** [review]

**Pull request merged**